packages/components/Menu/src/MenuItem/useMenuItem.ts

```diff
@@ -99,12 +114,23 @@ function getAccessibilityStateWorker(disabled: boolean, accessibilityState?: Acc
   return { disabled };
 }
 
-export const useHoverFocusEffect = (hovered: boolean, componentRef: React.MutableRefObject<any>) => {
+export const useHoverFocusEffect = (hovered: boolean, componentRef: React.MutableRefObject<any>, setIsMenuItemHovered: any) => {
```

Comment:
type of setIsMenuItemHovered should probably be defined instead of using `any`

same for onHoverOut.